tiered-storage/jcloud/src/main/java/org/apache/bookkeeper/mledger/offload/jcloud/impl/BlobStoreManagedLedgerOffloader.java

```diff
@@ -589,26 +299,17 @@ BlobStore getReadBlobStore(Map<String, String> offloadDriverMetadata) {
         return promise;
     }
 
-    public interface VersionCheck {
-        void check(String key, Blob blob) throws IOException;
-    }
-
+    
     @Override
     public OffloadPolicies getOffloadPolicies() {
-        return offloadPolicies;
+        // TODO Auto-generated method stub
+        return null;
     }
```

Comment:
Currently, when the `LedgerOffloader` is generated by the `PulsarService`, the `PulsarService` will check the namespace's `OffloadPolicies`, if the new `OffloadPolicies` is same as the existing `OffloadPolicies`, the `PulsarService` will reuse the existing `LedgerOffloader`.

Reply:
The focus of this PR is to refactor the tiered storage to use JCloud. I feel that we need to keep this PR to just that. I suggest we create an issue for this and assign it to me. I will make the change after this PR has been successfully merged.